src/Game.qml

```diff
@@ -294,13 +294,17 @@ Item {
     }
 
     Item {
+        property int length: parent.width < parent.height ? parent.width : parent.height
```

Comment:
That's reasonable but I would have replaced length with Dims.l(100) https://github.com/AsteroidOS/qml-asteroid/blob/master/src/controls/qml/Dims.qml#L68 that's what it was implemented for